test/test_utils/__init__.py

```diff
@@ -443,6 +443,17 @@ def is_dlc_cicd_context():
     return os.getenv("BUILD_CONTEXT") in ["PR", "CANARY", "NIGHTLY", "MAINLINE"]
 
 
+def is_not_dlc_image_type():
+    try:
+        return os.getenv("NOT_DLC").lower() == "true"
```

Comment:
can we change this to something like "IS_GENERIC_IMAGE" - I often find negatives in variable names that hold boolean values are confusing

Reply:
Included, thanks!!